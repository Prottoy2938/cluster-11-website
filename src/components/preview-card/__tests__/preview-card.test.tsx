import React from "react";
import { render, cleanup } from "@testing-library/react";
import PreviewCard from "../preview-card";

afterEach(cleanup);

test("should take and match snapshot", () => {
  const { asFragment } = render(
    <PreviewCard
      title="App Name"
      description="Description here"
      image="/imageRecognition/imageRecognition.jpg"
      demoURL="https://cluster-11.github.io/image-recognition-mobileNet/"
    />
  );
  expect(
    asFragment(
      <PreviewCard
        title="App Name"
        description="Description here"
        image="/imageRecognition/imageRecognition.jpg"
        demoURL="https://cluster-11.github.io/image-recognition-mobileNet/"
      />
    )
  ).toMatchSnapshot();
});
