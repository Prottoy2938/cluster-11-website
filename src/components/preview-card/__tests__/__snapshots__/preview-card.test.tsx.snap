// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take and match snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <div
      class="MuiPaper-root MuiCard-root root MuiPaper-elevation1 MuiPaper-rounded"
    >
      <div
        class="MuiCardHeader-root titleContainer"
      >
        <div
          class="MuiCardHeader-content"
        >
          <span
            class="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
          >
            App Name
          </span>
        </div>
      </div>
      <div
        class="MuiCardMedia-root media"
        style="background-image: url(/imageRecognition/imageRecognition.jpg);"
      />
      <div
        class="MuiCardContent-root"
      >
        <p
          class="MuiTypography-root descriptionContainer MuiTypography-body2 MuiTypography-colorTextSecondary"
        >
          Description here
        </p>
      </div>
      <div
        class="MuiCardActions-root actionContainer MuiCardActions-spacing"
      >
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-contained demoBtn"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-label"
          >
            Demo
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
        <div
          class="learnMoreContainer"
        >
          More Details
          <button
            aria-label="learn-more"
            class="MuiButtonBase-root MuiIconButton-root learnMoreIcon"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiIconButton-label"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M16.01 11H4v2h12.01v3L20 12l-3.99-4z"
                />
              </svg>
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
